src/applications/caregivers/containers/App.jsx

```diff
@@ -1,10 +1,35 @@
 import React from 'react';
+import { connect } from 'react-redux';
+
+import LoadingIndicator from '@department-of-veterans-affairs/formation-react/LoadingIndicator';
+
+import { toggleValues } from 'platform/site-wide/feature-toggles/selectors';
+import FEATURE_FLAG_NAMES from 'platform/utilities/feature-toggles/featureFlagNames';
 
 import RoutedSavableApp from 'platform/forms/save-in-progress/RoutedSavableApp';
 import formConfig from '../config/form';
 
-export default ({ location, children }) => (
-  <RoutedSavableApp formConfig={formConfig} currentLocation={location}>
-    {children}
-  </RoutedSavableApp>
-);
+const App = ({ loading, isFormAvailable, location, children }) => {
```

Comment:
We may want to consider adding prop types for these 🙂

Reply:
I added prop types form the props I added with the feature flag ✔️